pkg/internal/checkup/checkup.go

```diff
@@ -288,6 +288,7 @@ func generateBootScript() string {
 	sb.WriteString("  tuned_conf=\"/etc/tuned/realtime-virtual-guest-variables.conf\"\n")
 	sb.WriteString("  echo \"isolated_cores=" + isolatedCores + "\" > \"$tuned_conf\"\n")
 	sb.WriteString("  echo \"isolate_managed_irq=Y\" >> \"$tuned_conf\"\n")
+	sb.WriteString("  systemctl restart tuned.service\n")
```

Comment:
> What is the difference between the real-time and DPDK checkup with regard to this change?

This change should probably be done on DPDK as well. We didn't see the problem there but it may be only due to timing consideration..

>Typo?

Fixed typo.

Reply:
If this command is async, we should wait for it to finish.